apps/public-docsite-v9/src/Concepts/Upgrade/FromV0/migrate-custom-accessibility.stories.mdx

```diff
@@ -0,0 +1,109 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Concepts/Upgrading/from v0/Custom Accessibility" />
+
+# Migrate Custom Accessibility
+
+This guide provides instructions for migrating custom accessibility applied on stardust components to v9 components.
+
+## Migrate aria attributes
+
+Aria attributes should be kept during migration. They can be applied as props to the new component directly.
+
+## Migrate custom `accessibility` prop
+
+Custom `accessibility` prop has 4 properties:
+
+- `attributes`, contains aria attributes that can be applied directly to v9 components.
+- `keyActions` and `childBehaviors` are specific to @fluentui/react-northstar. The functionality is usually included in the components, for special cases contact Fluent UI team.
+- `focusZone` adds arrow key navigation to the container and child components. Consider replacing them with [react-tabster](https://github.com/microsoft/fluentui/blob/master/packages/react-tabster)
+
+## Migrate arrow key navigation
+
+Before:
+
+```tsx
+import { Grid, gridBehavior } from '@fluentui/react-northstar';
+const Component = () => (
+  <Grid accessibility={gridBehavior} role="menu">
+    <Button role="menuitem" content="First" />
+    ...
+  </Grid>
+);
+```
+
+After:
+
+```tsx
+import { Grid } from '@fluentui/react-components';
```

Comment:
There isn't a `Grid` control in `react-components`. Ideally examples should always use actual components and the code should be more or less copy-pasteable.

Reply:
good catch. There is a shim in TMP that is not yet here. will fix.